src/app/clients/form/birthYearDropdown.ts

```diff
@@ -1,21 +1,25 @@
-const currentDate = new Date();
-const currentYear = currentDate.getFullYear();
+const getCurrentYear = (): number => {
+    const currentDate = new Date();
+    return currentDate.getFullYear();
+};
 
-const childBirthYears: string[] = Array.from({ length: 17 }, (_, index) => {
-    return (currentYear - index).toString();
-});
+export const getChildBirthYears = (): string[] => {
+    const currentYear = getCurrentYear();
+    const childBirthYears: string[] = Array.from({ length: 17 }, (_, index) => {
+        return (currentYear - index).toString();
+    });
+    return childBirthYears;
+};
 
-export const childBirthYearList: [string, string][] = childBirthYears.map((year) => {
-    return [`${year}`, `${year}`];
-});
+export const getAdultBirthYears = (): string[] => {
+    const currentYear = getCurrentYear();
+    const adultBirthYears: string[] = Array.from({ length: 120 }, (_, index) => {
+        return (currentYear - index - 16).toString();
+    });
+    return adultBirthYears;
+};
 
-export const adultBirthYears: string[] = Array.from({ length: 120 }, (_, index) => {
-    return (currentYear - index - 16).toString();
-});
-
-export const adultBirthYearList: [string, string][] = adultBirthYears.map((year) => {
-    return [`${year}`, `${year}`];
-});
+export const youngestAdultBirthYear = getAdultBirthYears()[0];
```

Comment:
let's make this a function so that it gets computed every time instead of relying on the value at the build time.
```suggestion
export const getYoungestAdultBirthYear = () => getAdultBirthYears()[0];
```